src/app/pages/ArticlePage/Byline/index.tsx

```diff
@@ -104,6 +109,19 @@ const Byline = ({ blocks, children }: PropsWithChildren<Props>) => {
         {articleInformationTranslated}
       </VisuallyHiddenText>
       <ul css={BylineCss.bylineList} role="list">
+        {image ? (
+          <React.Fragment>
+            {isRtl ? (
+              <li css={BylineCss.imageRtl}>
+                <img css={BylineCss.imageSource} src={image} alt="" />
+              </li>
+            ) : (
+              <li css={BylineCss.imageLtr}>
+                <img css={BylineCss.imageSource} src={image} alt="" />
+              </li>
+            )}
+          </React.Fragment>
```

Comment:
Could probably just check if its `isRTL` in the `css` prop, rather than having a conditional render.

Not sure if its possible here, but can the layout of the whole Byline be achieved with flexbox? It should take care of the alignment in RTL without having to use floats. May overcomplicate things though since this needs to be in a ul/li.

```suggestion
          <li css={isRtl ? BylineCss.imageRtl : BylineCss.imageLtr}>
              <img css={BylineCss.imageSource} src={image} alt="" />
          </li>
```

Reply:
Good shout on checking `isRtl` within `css` prop. I did consider using flexbox and also grid but float worked well for this design

Reply:
There is some guidance in the component standards that might help with this https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/Styles.md#ltrrtl-design

I would also advise using flexbox but if you go with floats then you could use logical CSS properties for the spacings e.g. `margin-inline-end`